libraries/AC_Fence/AC_Fence.h

```diff
@@ -43,6 +44,8 @@ class AC_Fence
 
     void init() {
         _poly_loader.init();
+        AP_Param::set_and_save_by_name("FENCE_ALT_MAX", AC_FENCE_ALT_MAX_DEFAULT);
+        AP_Param::set_and_save_by_name("FENCE_ALT_FRAME", AC_FENCE_ALT_FRAME_DEFAULT);
```

Comment:
This definitely doesn't look right.  What happened to people's persistence here?!